### PR TITLE
add clean CoC + response

### PR DESCRIPTION
**Pull request by Bradamant3**
Sorry folks, made a real mess out of last PR for CoC response. This contains changes from PR 115 and 122 for the CoC, plus new CoC response. Note that I've implemented Erik's suggestion of a conduct@writethedocs.org email alias; not set in stone, but gives us a starting point. No phone numbers; we'll have to discuss how to manage that one.

### CODE REVIEW

File: docs/code-of-conduct-response.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+* If this is determined to be an ongoing incident or a threat to physical safety, the staff's immediate priority will be to protect everyone involved. This means we may delay an "official" response until we believe that the situation has ended and that everyone is physically safe.
+
+After the staff has a complete account of the events they will make a decision as to how to response. Responses may include:
+
+* Nothing (if we determine no violation occurred).
+* A private reprimand from the staff to the individual(s) involved.
+* A public reprimand.
+* An imposed vacation (for example, asking someone to "take a week off" from Slack or the forum).
+* A permanent or temporary ban from some or all WTD spaces (Slack, the forum, meetups).
+* A request for a public or private apology.
```
Comment (mjang) [inline docs/code-of-conduct-response.rst:47]: Since this seems to be in order of severity, I'd put the "apology" line before "A public reprimand"

**Review by mjang — APPROVED** [review]
LGTM with a couple of minor comments. Once merged, I think we'll want a "short version" for Meetups.

Hint: if the revs are short, I'd make the revs directly in this GH web interface.

**Comment by ericholscher** [comment]
The reporting guide is a good thing for attendees for our events, but I don't know if we should remove our internal response docs. We need to have a standard way of dealing with things internally, which is more what the response docs were about. This feels like 3 different things:

* The COC which specifies the rules
* The reporting guide, which explains in more detail to attendees how to report issues
* A response guide, which is used internally to make sure people who receive the reports handle it properly.

If we wanted to update our response guide, we can steal that again from Django: https://github.com/django/code-of-conduct/blob/master/reports.md -- but it feels like a necessary part of this, which shouldn't be deleted.

File: docs/code-of-conduct-response.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
+
+Write the Docs Code of Conduct - Reporting Guide
+================================================
+
+.. _report: conduct@writethedocs.org
+
+In general, in any Write the Docs space (conferences, meetup, Slack, forum), if you believe someone is violating the code of conduct we ask that you report it to the staff by emailing report_. All reports will be kept confidential. In some cases we may determine that a public statement will need to be made. If that's the case, the identities of all victims and reporters will remain confidential unless those individuals instruct us otherwise.
```
Comment (plaindocs) [inline docs/code-of-conduct-response.rst:7]: How about 

> if you believe someone is violating the code of conduct,  in any Write the Docs space (conferences, meetup, Slack, forum), please (or we ask that...) report it to the staff by emailing report_

File: docs/code-of-conduct.rst [hunk 3]
```diff
@@ -0,0 +1,34 @@
+
+Code of Conduct
+===============
+
+Write the Docs is a global community of documentarians who share information, discuss ideas, and work together to improve the art and science of documentation.
+
+Diversity is one of our huge strengths. To support a welcoming environment for all, regardless of individual differences, we have a few ground rules that we ask people to adhere to when they're participating within this community and its projects. These rules apply equally to founders, mentors, admins, and those seeking conversation, discussion, help, and guidance -- in short, to all participants.
```
Comment (plaindocs) [inline docs/code-of-conduct.rst:7]: I'd say "in this community" not "within this community"

**Review by plaindocs — APPROVED** [review]

File: docs/code-of-conduct-response.rst (hunk 2 above)
Comment (yo-bj) [inline docs/code-of-conduct-response.rst:7]: If the report includes a minor, there is a chance that Mandatory Reporting laws would apply; therefore, we might want to clarify the "All reports will be kept confidential" to reflect our legal obligation.

File: docs/code-of-conduct-response.rst (hunk 1 above)
Comment (yo-bj) [reply docs/code-of-conduct-response.rst:47]: Seems reasonable. +1

File: docs/code-of-conduct-response.rst [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+* A public reprimand.
+* An imposed vacation (for example, asking someone to "take a week off" from Slack or the forum).
+* A permanent or temporary ban from some or all WTD spaces (Slack, the forum, meetups).
+* A request for a public or private apology.
+
+We'll respond within one week to the person who filed the report with either a resolution or an explanation of why the situation is not yet resolved.
+
+Once we've determined our final action, we'll contact the original reporter to let them know what action we'll be taking, if any. We'll take into account feedback from the reporter on the appropriateness of our response, but we don't guarantee we'll act on it.
+
+Finally, the staff may choose to make a public report of the incident.
```
Comment (yo-bj) [inline docs/code-of-conduct-response.rst:53]: What are the criteria for making the report public?

File: docs/code-of-conduct-response.rst [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+Appealing
+---------
+
+Only permanent resolutions (such as bans) may be appealed. To appeal a decision, contact the WTD staff at report_ with your appeal and they will review the case.
```
Comment (yo-bj) [inline docs/code-of-conduct-response.rst:58]: Will the same people who handled the initial report be asked to do the appeal as well? I would ask that we consider bringing in a separate set of folks in for appeals.

**Review by ericholscher — CHANGES_REQUESTED** [review]
The reporting guide is a good thing for attendees for our events, but I don't know if we should remove our internal response docs. We need to have a standard way of dealing with things internally, which is more what the response docs were about. This feels like 3 different things:

The COC which specifies the rules
The reporting guide, which explains in more detail to attendees how to report issues
A response guide, which is used internally to make sure people who receive the reports handle it properly.
If we wanted to update our response guide, we can steal that again from Django: https://github.com/django/code-of-conduct/blob/master/reports.md -- but it feels like a necessary part of this, which shouldn't be deleted.

File: docs/code-of-conduct-response.rst (hunk 4 above)
Comment (mjang) [reply docs/code-of-conduct-response.rst:53]: If it comes to that, I'd ask for legal advice. But for the CoC, I might include an `if appropriate` clause, e.g.: "Finally, if appropriate, the staff may choose to make a public report of the incident."

File: docs/code-of-conduct-response.rst (hunk 5 above)
Comment (mjang) [reply docs/code-of-conduct-response.rst:58]: Maybe -- I think judges sometimes reverse their own rulings on appeal (after sleeping on it)

File: docs/code-of-conduct-response.rst (hunk 4 above)
Comment (yo-bj) [reply docs/code-of-conduct-response.rst:53]: Ah, my "rules lawyer" brain is asking for criteria for the public to see. "If appropriate" is a step forward, though very vague. How about this: "Finally, depending on several factors surrounding the incident (scope, seriousness, legal obligations, etc.), the staff reserve the right to make a public report of the incident."

**Comment by thatdocslady** [comment]
So I was refactoring the CoC when I realized that this amazing discussion has been happening while I wasn't being attentive. Now I see that there's a bunch of feedback that wasn't all addressed. @Bradamant3 are you planning to handle this soon? I also have some additions I wanted to make (which is why I refactored originally), so maybe we can merge this and open a follow-up issue for the open issues, since it looks to me largely resolved/awesome :)

**Comment by thatdocslady** [comment]
Ok I vote to merge this PR for the sake of preserving the great additions that were made here, and then I will take my input from the PR I submitted #146 as well as some of the feedback suggestions from here and submit a new PR on top of these merged changes. Unless anyone has objections to this, I will do it tomorrow (it's late CET :).

**Comment by Bradamant3** [comment]
Go Mikey. I have neglected it for too long, so sorry! Thank you!

**Pull request merged**